My events tab: view and manage your events (non-admin)

samzx opened the pull request:
## Short description
resolves #46  - Ability for users to see past events

## Description
- Changed the "Individual Events" tab to be "My Events". This should reflect it's new functionality.
- Added the ability to see events (non-individual events) that users have attended in the past. Also gives a user a sense of accomplishment.
- Refactored the individual events table to use `ReactTable` to allow sorting by a column. This is useful to display the latest events first.

## References
[Github Issue](https://github.com/zendesk/volunteer_portal/issues/46)

## Screenshots (if needed)

### Before

<details>

![image](https://user-images.githubusercontent.com/27116427/56639562-4e945080-66b4-11e9-9641-ea8aee86c1f9.png)


</details>

#### After

<details>

![image](https://user-images.githubusercontent.com/27116427/56639439-0b39e200-66b4-11e9-887c-fc54da0df392.png)

</details>

## CCs

@zendesk/volunteer

samzx committed:
added react table view of organized events

samzx committed:
updated individual events table with react tables

samzx committed:
make actions column unsortable

samzx committed:
fix white space

xuebingli commented on app/javascript/pages/IndividualEvents/index.js:
```diff
@@ -261,86 +262,172 @@ const DestroyDialog = ({ onDelete, onCancel, popover }) => (
+  {
+    id: 'location',
+    Header: 'Location',
+    accessor: d => d.location,
```
I believe
```suggestion
    accessor: 'location',
```
works, too. A bit shorter.

samzx replied:
ah yes thanks for that!

xuebingli reviewed (approved):
👏 Well done, Sam!

samzx committed:
refactored customer react table accessor to string based for easier readability

xuebingli commented on app/javascript/pages/IndividualEvents/query.gql:
```diff
@@ -2,7 +2,7 @@
 #import "fragments/OrganizationEntry.gql"
 
-query IndividualEventsQuery {
+query MyEventsQuery {
```
Since this query is not only about individual events, should we move/rename this file?

xuebingli commented on app/javascript/pages/IndividualEvents/query.gql:
```diff
@@ -22,6 +22,24 @@ query IndividualEventsQuery {
     }
+    signups {
+      event {
+        id
```
Is this field used?

samzx replied:
nope, you're right! It must've snuck in when I was testing, I'll remove this

samzx committed:
updates based on pr feedback
- remove unused id fields
- renamed page and routes from individual events to my events to better match what is displayed to users
- added margins to bottom of table for white space

samzx committed:
add comment to keep old route

xuebingli commented:
👍
